### PR TITLE
Upgrade cosmos to commit f1fdde5d1b18

### PR DESCRIPTION
**Pull request by taobun**
We bump cosmos-sdk version to the same version of the current relayer to make sure we can communicate via ibc again.

### CODE REVIEW

File: chain/app/app.go [hunk 1]
```diff
@@ -183,57 +184,54 @@ func NewBandApp(
...
+		appCodec, keys[auth.StoreKey], app.subspaces[auth.ModuleName], auth.ProtoBaseAccount, maccPerms,
 	)
 	app.BankKeeper = bank.NewBaseKeeper(
 		appCodec, keys[bank.StoreKey], app.AccountKeeper, app.subspaces[bank.ModuleName], app.ModuleAccountAddrs(),
 	)
-	app.SupplyKeeper = supply.NewKeeper(
-		appCodec, keys[supply.StoreKey], app.AccountKeeper, app.BankKeeper, maccPerms,
-	)
 
 	// Wrapped supply keeper allows burned tokens to be transferred to community pool
```
Comment (sorawit) [inline chain/app/app.go:206]: Add TODO

**Review by sorawit — CHANGES_REQUESTED** [review]

File: chain/app/app.go [hunk 2]
```diff
@@ -392,14 +400,14 @@ func (app *BandApp) InitChainer(ctx sdk.Context, req abci.RequestInitChain) abci
...
+	return app.LoadVersion(height)
 }
 
 // ModuleAccountAddrs returns all the app's module account addresses.
 func (app *BandApp) ModuleAccountAddrs() map[string]bool {
 	modAccAddrs := make(map[string]bool)
 	for acc := range maccPerms {
-		modAccAddrs[supply.NewModuleAddress(acc).String()] = true
+		modAccAddrs[auth.NewModuleAddress(acc).String()] = true
 	}
```
Comment (Benzbeeb) [inline chain/app/app.go:412]: please add
```
ctx := app.BaseApp.NewContext(true, abci.Header{})
app.capabilityKeeper.InitializeAndSeal(ctx)
```

File: chain/app/app.go [hunk 3]
```diff
@@ -311,15 +313,21 @@ func NewBandApp(
...
+		),
+	)
 	app.SetEndBlocker(app.EndBlocker)
 
 	if loadLatest {
-		err := app.LoadLatestVersion(app.keys[bam.MainStoreKey])
+		err := app.LoadLatestVersion()
 		if err != nil {
 			tmos.Exit(err.Error())
 		}
```
Comment (Benzbeeb) [inline chain/app/app.go:333]: please add at line 335
```
ctx := app.BaseApp.NewContext(true, abci.Header{})
app.capabilityKeeper.InitializeAndSeal(ctx)
```
```

Comment (Benzbeeb) [inline chain/app/app.go:334]: add this at line 335
```
ctx := app.BaseApp.NewContext(true, abci.Header{})
app.capabilityKeeper.InitializeAndSeal(ctx)
```

**Review by Benzbeeb — CHANGES_REQUESTED** [review]

**Review by sorawit — APPROVED** [review]

**Review by Benzbeeb — APPROVED** [review]

**Pull request merged**